include/mbedtls/ecdh.h

```diff
@@ -127,6 +166,16 @@ int mbedtls_ecdh_compute_shared( mbedtls_ecp_group *grp, mbedtls_mpi *z,
  */
 void mbedtls_ecdh_init( mbedtls_ecdh_context *ctx );
 
+/**
+ * \brief           This function sets up the ECDH context.
```

Comment:
Please add a paragraph explaining what this means and how this fits in the lifecycle of an ECDH context object (e.g. should I call `init` first?).